### ISSUE TITLE
Sub content under the collapsed button is getting announced in collapsed state

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4576]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, Role, Value

**Test Environment:**
- Test URL: https://clarity.design/documentation/stack-view
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto one of the collapsed stack view sections (except for static stack-view) and observe the spoken feedback of screen reader.

**Expected result:**
Appropriate announcement made should be made for the collapsed button i.e. Label, control type, state and any other relevant information should be communicated when any one of the collapsed stack view buttons are focused.The underlying collapsed content should be announced only after the content is exposed by expanding the button.
In this scenario, focus should be placed on the first sub label and appropriate announcement should be made.

**Actual result:**
Label, control type and current state of the collapsed buttons are announced appropriately along with a redundant hint. But along with that underlying collapsed content is also getting announced. 

**Note:** It's not failing WCAG 4.1.2, but it's blocking it in a way by announcing the collapsed content along with Name, role and state.
